website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,190 @@
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-cmd-status.jpg" width="100%" title="Use the command bar to write dbt commands and the status button for the current IDE status"/>
+
+
+## Editing features
+
+The IDE features some delightful tools and layouts to make it easier for you to write dbt code and collaborate with teammates. 
+
+- **File Editor &mdash;** The File Editor is where users edit code. Tabs break out the region for each opened file, and unsaved files are marked with a blue dot icon in the tab view.
+
+- **Format button &mdash;** The editor has a **Format** button for `{{ filetypes }}`files. When you click the button, the editor will use `sqlfmt` or `sqlfluff` for SQL files and `black` for Python files to reformat the contents of the file.
+
+- **Save button &mdash;** The editor has a **Save** button that saves editable files. Pressing the button or using the Command-S or Control-S shortcut saves the file contents. Saving isn't needed to preview code results in the Console section, but it's necessary before changes appear in a dbt invocation. The File Editor tab shows a blue icon for unsaved changes.
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-editing.jpg" width="100%" title="Use the file editor, format, and save button during your development workflow"/>
+
+## Console
+The console section, located below the File editor, includes various console tabs and buttons to help you with tasks such as previewing, compiling, building, and viewing the DAG. Refer to the following sub-bullets for more details on the console tabs and buttons.
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-console.jpg" width="100%" title="Use the file editor, format, and save button during your development workflow"/>
+
+- **Preview console button &mdash;** When you click on the Preview Button, it runs the SQL in the active file editor regardless of whether you have saved it or not, and sends the results to the Results console tab. To prevent the IDE from returning too much data and causing browser problems, a limit of 500 is automatically added to queries executed via the Preview Button.
+
+- **Compile console button &mdash;** The Compile Button compiles the SQL code from the active File Editor, irrespective of its save status, and outputs it to the Compiled Code console tab.
+
+- **Build console button &mdash;** The build button allows users to quickly access dbt commands related to the active model in the File Editor. The available commands include dbt build, dbt test, and dbt run, with options to include only the current resource, the resource and its upstream dependencies, the resource and its downstream dependencies, or the resource with all dependencies. This menu is available for all executable nodes.
+
+- **Results console tab &mdash;** The Results console tab displays the most recent Preview results in tabular format. 
```

Comment:
hey @dave-connors-3- i'm sure this is dummy data but wanted to triple check that the results here aren't real results? there is a phone num column so wanted to be super sure

Reply:
hey @dave-connors-3- i'm sure this is dummy data but wanted to triple check that the results here aren't real results? there is a phone num column so wanted to be super sure